library/bignum_core.h

```diff
@@ -60,6 +60,57 @@ size_t mbedtls_mpi_core_bitlen( const mbedtls_mpi_uint *A, size_t A_limbs );
 void mbedtls_mpi_core_bigendian_to_host( mbedtls_mpi_uint *A,
                                          size_t A_limbs );
 
+/**
+ * \brief   Perform a safe conditional copy of MPI which doesn't reveal whether
+ *          the condition was true or not.
+ *
+ * \param[OUT] X        The address of the first MPI. This must be initialized.
+ *                      It must have at least \p limbs limbs.
+ * \param[IN]  A        The address of the second MPI. This must be initialized.
```

Comment:
We use lowercase `in` and `out` everywhere that I can see
```suggestion
 * \param[out] X        The address of the first MPI. This must be initialized.
 *                      It must have at least \p limbs limbs.
 * \param[in]  A        The address of the second MPI. This must be initialized.
```